File: Python/7.reverseInteger.py
class Solution:
    def reverse(self, x: int) -> int:
        num = 0
        a = abs(x)
        while(a != 0):
            temp = a % 10
            num = num * 10 +temp
            a = a // 10
        if x > 0 and num < 2147483647:
            return num
        elif x < 0 and num <= 2147483647:
            return -num
        else:
            return 0

File: Javascript/7.reverseInteger.js
var reverse = function(x) {
    let num = parseInt(x.toString().split('').reverse().join(''));
    if (num > Math.pow(2, 31)) {
        return 0;
    }
    if (x < 0) {
        return num * (-1);
    }
    else {
        return num;
    }
};